pkg/agent/client/client_test.go

```diff
@@ -28,13 +27,68 @@ var (
 	log, _ = test.NewNullLogger()
 
 	trustDomainURL = url.URL{Scheme: "spiffe", Host: "example.org"}
+
+	testEntries = []*common.RegistrationEntry{
```

Comment:
These test structures were ripped out of the newTestFetchX509SVIDResponse() function, which had them packed into a node API response struct (the removal of that function is below)